v2/dtos/responses/event.go

```diff
@@ -14,8 +14,7 @@ import (
 // This object and its properties correspond to the AddEventResponse object in the APIv2 specification:
 // https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/AddEventResponse
 type AddEventResponse struct {
-	common.BaseResponse `json:",inline"`
-	ID                  string `json:"id"`
+	common.BaseWithIdResponse `json:",inline"`
```

Comment:
Remove AddEventResponse  and just use common.BaseWithIdResponse. Update swagger to match

Reply:
Removed the AddEventResponse and updated the swagger.